src/components/CurrentTask/CurrentTask.js

```diff
@@ -1,14 +1,29 @@
 import useStore from '../../hooks/useStore';
+import Button from '../Button/Button';
 
 import StyledCurrentTask from './StyledCurrentTask';
 
 export default function CurrentTask() {
 	const tasks = useStore(state => state.tasks);
+	const deleteTask = useStore(state => state.deleteTask);
```

Comment:
If you want to implement a delete-function in the future (for non-completed tasks), it would be better to rename this function to avoid confusion. The user is marking the exercise as "done" even though you delete the task from tasks array. So you could also name it something like "completeTask" I guess 🤔